app/controllers/api/services_controller.rb

```diff
@@ -25,6 +25,57 @@ def reconfigure_resource(type, id = nil, data = nil)
       end
     end
 
+    def start_resource(type, id = nil, _data = nil)
+      raise BadRequestError, "Must specify an id for starting a #{type} resource" unless id
+
+      api_action(type, id) do |klass|
+        service = resource_search(id, type, klass)
+        api_log_info("Starting #{service_ident(service)}")
+
+        begin
+          description = "#{service_ident(service)} starting"
+          task_id = queue_object_action(service, description, :method_name => "start", :role => "ems_operations")
```

Comment:
@imtayadeway - Yes, that looks correct.  

In the service model we call `queue_group_action` when calling `start`, `stop`, etc. directly on a service.  https://github.com/ManageIQ/manageiq/blob/master/app/models/service.rb#L172-L186

You'll see on line 177 the role `ems_operations` is hard coded.